### ISSUE TITLE
Size of input vector

### ISSUE THREAD

**Issue by Prabal1998** [matousc89/padasip#26]
Could you please help to know if input vector should be resized into fixed batch size during training? 

-----------Error observed in the following setup-----------
Setup:
1.Input vector of shape:(140928,)
2.Length of filter: 16
3. Desired vector of shape: (140928,)

### In step 3:   After resizing either row or column changes size. 
###This contradicts either of the requirement
 ###1. Number of rows equals to number of input shape 
 ###3. Number of columns equals to the length of filter.
###4. length of input vector and desired vector 

4. Resizing the input vector: 1. Transpose 2. Resize into (140928, 140928/16)

Observation:
In run() method following 

![image](https://github.com/matousc89/padasip/assets/40731891/72142110-eb22-417a-ae0c-b486f175193d)

So basically, I presume that before feeding input vector directly, it is necessary to resize  into batch size meeting the requirement of step 3.  
OR,
What is a correct approach to carry out the simulation?

**Comment by Prabal1998** [comment]
On further exploring, two dimensional arrays of delayed samples are obtained using methods pa.input_from_history(). Argument and implementation are similar to those of transversal filter approach.
